http3/server.go

```diff
@@ -313,6 +313,15 @@ func extractPort(addr string) (int, error) {
 	return portInt, nil
 }
 
+func findString(haystack []string, needle string) int {
```

Comment:
You're not using the `int` return value. What about `contains([]string, string) bool` instead?

Reply:
Or just inline the entire function, then we don't need to think about naming ;)